lib/Notification/bar/NotificationBarBase.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Alert from 'react-bootstrap/lib/Alert';
+import cx from 'classnames';
+import Button from '../../Button';
+import Icon from '../../Icon';
+import { defaults, types } from './barTypes';
+
+/**
+ * @usage
+ *
+ * <NotificationBarBase>This is a warning</NotificationBarBase>
+ */
+const NotificationBarBase = ({
+  children,
+  clickHandler,
+  className,
+  center,
+  onClose,
+  ...rest
+}) => {
+  const classes = cx({
+    'cursor-pointer': clickHandler,
+    'text-center': center,
+    flex: onClose,
+    'justify-end': center && onClose,
+    'notification--has-close': onClose
```

Comment:
🚩 Do we need this class anymore? I _think_ that it was removed from the styles, and line 24 does what it used to.